src/utils/validation.js

```diff
@@ -109,6 +111,9 @@ const editCourseValidate = (values, props) => {
           }
         }
       });
+      if (Object.keys(runErrors).length > 0) {
```

Comment:
`Object.keys(runErrors).length` would be fine.  if length is 0 it evaluates to false anyway